Scripts/RemediationScripts/Remediate-EnableHttpsForStorageAccount.ps1

```diff
@@ -0,0 +1,723 @@
+﻿<##########################################
+
+# Overview:
+    This script is used to make the transit in Storage Account encrpyted using the HTTPS.
```

Comment:
Typo: `encrpyted` to be changed to `encrypted`.

Reply:
Done